src/sider/senoppfolging/SenOppfolging.tsx

```diff
@@ -25,22 +27,24 @@ const texts = {
 
 export default function SenOppfolging(): ReactElement {
   const { data: kandidater } = useSenOppfolgingKandidatQuery();
-  const kandidat = kandidater[0];
+  const kandidat: SenOppfolgingKandidatResponseDTO | undefined = kandidater[0];
```

Comment:
Bare en tanke – Her ser det ut som vi kunne hatt en query parameter for `/senoppfolging/kandidater` endepunktet for å hente sist gang vedkommende var kandidat 😊